tsl/test/expected/dist_api_calls.out

```diff
@@ -380,13 +380,4 @@ SELECT approximate_row_count('disttable_repl');
 (1 row)
 
 ANALYZE disttable_repl;
-SELECT approximate_row_count('disttable_repl');
- approximate_row_count 
------------------------
-                     8
-(1 row)
-
-DROP TABLE disttable_repl;
-DROP DATABASE :DN_DBNAME_1;
-DROP DATABASE :DN_DBNAME_2;
-DROP DATABASE :DN_DBNAME_3;
+ERROR:  [data_node_1]: relation "disttable_repl" does not exist
```

Comment:
Hmm that doesnt look right, and shouldnt be part of your PR. I think this is just result of flaky multinode test you accidentally added.